tmva/tmva/inc/TMVA/RBDT.hxx

```diff
@@ -48,6 +48,9 @@ public:
    {
       // Get number of output nodes of the forest
       auto file = TFile::Open(filename.c_str(), "READ");
```

Comment:
is the file leaked?
```suggestion
      std::unique_ptr<TFile> file{TFile::Open(filename.c_str())};
```

Reply:
Yes, correct. It was only closed but not deleted the file ! Thanks for finding this!